dev-util/visual-studio-code/visual-studio-code-1.22.2.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit check-reqs eutils xdg-utils
+
+DESCRIPTION="Microsoft's free and open-source code editor."
+HOMEPAGE="https://code.visualstudio.com/"
+SRC_URI="https://github.com/Microsoft/vscode/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://www.hlmjr.com/vscode-assets.tar.gz"
```

Comment:
What are those, why aren't they versioned, how are they created, where are they being used?

Reply:
It's the application icon. For whatever reason the newest application icon is in the Debian packaged binary version, but not in the source repository. I tarred it up and hosted it myself per a suggestion in the gentoo-proxy-maint channel.